src/autofill-utils.js

```diff
@@ -519,6 +519,48 @@ function safeRegexTest (regex, string) {
     return regex.test(string)
 }
 
+/**
+ * Returns the event target, or an element that matches wantedTargetType, piercing the shadow tree
+ * @param {PointerEvent | FocusEvent} event
+ * @param {typeof Element} [wantedTargetType]
+ * @returns {*|Element}
```

Comment:
I think this is like `any|Element`, it would be good to tighten that up (probably your editor made this). not a blocker at all

Reply:
Fixed. Thank you!